debug.log

```diff
@@ -0,0 +1 @@
+[1215/214541.718:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
@StephenAraka Bro whats' up with this?

Reply:
eh man... hadn't seen that. Must be some Windows created file... I need to take that out!